src/Views/ResizeView.vala

```diff
@@ -51,15 +51,15 @@ public class ResizeView : AbstractInstallerView {
         scale.inverted = true;
         
         scale.show_fill_level = true;
-        scale.get_style_context ().add_class (Granite.STYLE_CLASS_ACCENT);
+        scale.get_style_context ().add_class ("accent");
```

Comment:
Considering elementary OS is build from bionic, are you sure? Does Pop!_OS just have an outdated Granite?

Reply:
Builds are failing otherwise, so likely.

Reply:
Pop does not include Granite, so it uses the version in the Ubuntu repository: https://packages.ubuntu.com/bionic/libgranite-dev

Reply:
@mmstick ah. That may become a problem down the road with both the installer and AppCenter, so it might be worth copying Granite over from the elementary repo.

Reply:
Importing it now